FocalTech/FT5xx6/GHIElectronics.TinyCLR.Drivers.FocalTech.FT5xx6.csproj

```diff
@@ -42,22 +42,22 @@
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="GHIElectronics.TinyCLR.Devices.Gpio, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\GHIElectronics.TinyCLR.Devices.Gpio.1.0.0\lib\net452\GHIElectronics.TinyCLR.Devices.Gpio.dll</HintPath>
+    <Reference Include="GHIElectronics.TinyCLR.Devices.Gpio, Version=2.0.0-preview1, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Should be `<Reference Include="GHIElectronics.TinyCLR.Devices.Gpio, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">`, fix all projects